target/linux/ipq806x/files-5.15/arch/arm/boot/dts/qcom-ipq8068-ap3935.dts

```diff
@@ -0,0 +1,368 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq8064-v2.0.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+	model = "Extreme Networks AP3935";
+	compatible = "extreme,ap3935", "qcom,ipq8064";
+
+	memory@0 {
+		reg = <0x41400000 0x3ec00000>;
+		device_type = "memory";
+	};
+
+	cpus {
+		idle-states {
+			CPU_SPC: spc {
+				status = "disabled";
+			};
+		};
+	};
+
+	aliases {
+		serial0 = &gsbi7_serial;
+		serial1 = &gsbi2_serial;
+		mdio-gpio0 = &mdio0;
+		ethernet0 = &gmac0;
+		ethernet1 = &gmac2;
+
+		led-boot = &led_power_yellow;
+		led-failsafe = &led_power_orange;
+		led-running = &led_power_yellow;
+		led-upgrade = &led_power_yellow;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+		bootargs-override = "loglevel=8 earlycon=msm_serial_dm,0x16640000 console=ttyMSM0,115200n8 ubi.block=0,0 root=/dev/ubiblock0_0 rootfstype=squashfs";
```

Comment:
logelevel? earlycon? think this can be reduced and include only the root entry?

Reply:
I will delete all except "root"